app/src/main/res/values/dimens.xml

```diff
@@ -45,6 +45,13 @@
     <dimen name="settings_close_button_height">96dp</dimen>
     <dimen name="settings_close_button_width">112dp</dimen>
     <dimen name="settings_pointer_elevation">16dp</dimen>
+    
+    <dimen name="settings_handset_close_button_layout_width">72dp</dimen>
```

Comment:
Instead of making new dimens with different names, you should make a new dimens file for the tablet in a directory called `values-sw600dp`. Then copy the original values into the new `sw600dp` file and update the values in this file to match the handset designs.